.github/workflows/ci.yml

```diff
@@ -12,11 +12,13 @@ defaults:
   run:
     shell: bash
 
+permissions:
+  issues: write
+  pull-requests: write
```

Comment:
Would `pull-requests: write` mean the action can push commits to a PR?

I don't think these are particularly scary, but we might want to move the jobs which don't require this into a different workflow to avoid giving permissions where they aren't needed.

Reply:
Even better